### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by ISRLv2021**
- [DEMO LINK](https://ISRLv2021.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://ISRLv2021.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,53 @@
...
+        Код товара: 195434
+      </p>
+      <div class="card__stars-reviews">
+        <div class="stars stars--4">
+          <div class="star"></div>
+          <div class="star"></div>
+          <div class="star"></div>
+          <div class="star"></div>
+          <div class="star"></div>
+        </div>
```
Comment (ZUBOV-ILLIA) [inline src/index.html:29]: `star` should be element of star block

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
```
Comment (ZUBOV-ILLIA) [inline src/style.css:12]: `width: 200px;` you cant set width to BAM block

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,53 @@
...
+      </div>
+      <div class="card__price-container">
+        <div class="price">
+          Цена:
+        </div>
+        <div class="price-number">
+          69 999 грн
+        </div>
+      </div>
+      <div class="card__button-container">
```
Comment (ZUBOV-ILLIA) [inline src/index.html:42]: `<div class="card__button-container">` you not need here this container. Can remove.

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+  border-radius: 5px;
+  text-decoration: none;
+  transition: 1s;
+}
+
+.buy-button:hover {
+  background-color: white;
+  border: 1px solid #00acdc;
+  color: #00acdc;
+}
```
Comment (ZUBOV-ILLIA) [inline src/style.css:124]: You forget set border for `.buy-button`

**Review by ZUBOV-ILLIA — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (Anna-Tkachenko) [reply src/index.html:29]: agree, fix this comment
star should be element of the stars block

File: src/index.html [hunk 5]
```diff
@@ -1,13 +1,53 @@
...
+        <div class="stars stars--4">
+          <div class="star"></div>
+          <div class="star"></div>
+          <div class="star"></div>
+          <div class="star"></div>
+          <div class="star"></div>
+        </div>
+        <div class="review">
+          Отзывов: 5
+        </div>
```
Comment (Anna-Tkachenko) [inline src/index.html:32]: Use semantic tags

File: src/index.html [hunk 6]
```diff
@@ -1,13 +1,53 @@
...
+      </div>
+      <div class="card__price-container">
+        <div class="price">
+          Цена:
+        </div>
+        <div class="price-number">
+          69 999 грн
+        </div>
```
Comment (Anna-Tkachenko) [inline src/index.html:40]: Use semantic tags

File: src/index.html [hunk 7]
```diff
@@ -1,13 +1,53 @@
...
+      </div>
+      <div class="card__button">
+        <a
+          href="#"
+          class="buy-button"
+          data-qa="hover"
+        >
+          купить
+        </a>
+      </div>
```
Comment (Anna-Tkachenko) [inline src/index.html:50]: Don't create additional wrappers

File: src/style.css [hunk 8]
```diff
@@ -0,0 +1,125 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
```
Comment (Anna-Tkachenko) [inline src/style.css:5]: Set font-family to html tag

File: src/style.css [hunk 9]
```diff
@@ -0,0 +1,125 @@
...
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
```
Comment (Anna-Tkachenko) [inline src/style.css:10]: This styles are redundant

File: src/style.css (hunk 2 above)
Comment (Anna-Tkachenko) [reply src/style.css:12]: Agree, you can't set width for BEM block

File: src/style.css [hunk 10]
```diff
@@ -0,0 +1,125 @@
...
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  padding: 32px 16px 28px;
+}
+
+.card__logo {
+  width: 160px;
+  height: 134px;
+  margin-left: auto;
+  margin-right: auto;
```
Comment (Anna-Tkachenko) [inline src/style.css:23]: margin: 0 auto;

File: src/style.css [hunk 11]
```diff
@@ -0,0 +1,125 @@
...
+.review {
+  font-weight: 400;
+  color: #060b35;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.price {
+  font-size: 12px;
+  font-weight: 400;
```
Comment (Anna-Tkachenko) [inline src/style.css:68]: Don't create additional BEM blocks

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]

**Review by Anna-Tkachenko — APPROVED** [review]
Good job! 👍

**Pull request closed without merging**